sphere-models/src/test/resources/ProductProjectionTest/product-projection-from-search.json

```diff
@@ -0,0 +1,128 @@
+{
+  "id": "24593118-1354-442d-96b7-25f2a408ee41",
+  "version": 6,
+  "productType": {
+    "typeId": "product-type",
+    "id": "4698af68-fa24-49ab-8a3e-abb113d72c0a"
+  },
+  "name": {
+    "en": "Book",
+    "de": "Buch"
+  },
+  "categories": [],
+  "categoryOrderHints": {},
+  "slug": {
+    "de": "buch-1448540659781",
+    "en": "book-1448540608849"
+  },
+  "masterVariant": {
+    "id": 1,
+    "sku": "book-black-sku",
+    "prices": [
+      {
+        "value": {
+          "currencyCode": "EUR",
+          "centAmount": 1300
+        },
+        "id": "0a232258-0d95-4f18-883e-99551eacd545",
+        "channel": {
+          "typeId": "channel",
+          "id": "5724e113-86d8-4a4c-aa30-18dbc4ebcbf6"
+        }
+      },
+      {
+        "value": {
+          "currencyCode": "EUR",
+          "centAmount": 2000
+        },
+        "id": "15ca574c-ec3e-49b5-9de2-dbf32bee26b6"
+      }
+    ],
+    "images": [],
+    "attributes": [
+      {
+        "name": "color",
+        "value": "black"
+      },
+      {
+        "name": "longDescription",
+        "value": {
+          "en": "Some long description for the book",
+          "de": "Some long description for the book in german"
+        }
+      }
+    ],
+    "isMatchingVariant": false
```

Comment:
I don't have strong opinion about the naming of this attribute, unfortunately it would be a breaking change now.

We can't really know who is using it so we would have to go through a deprecation process.
@mmoelli WDYT?